packages/host/app/components/operator-mode/submode-layout.gts

```diff
@@ -256,7 +256,7 @@ export default class SubmodeLayout extends Component<Signature> {
         margin-right: 0;
         background-color: var(--boxel-ai-purple);
         box-shadow: var(--boxel-deep-box-shadow);
-        z-index: calc(var(--boxel-modal-z-index) - 1);
+        z-index: calc(var(--boxel-modal-z-index) - 2);
```

Comment:
I had to change this because the ai button should be hidden behind the overlay. 

modal z index: 15
overlay z index: 15-1
ai chat btn: 15-2